### ISSUE TITLE
'Unlock wallet first' error message needs to be in plain English

### ISSUE THREAD

**Issue by sysmca** [syscoin/blockmarket-desktop-public#174]
Any time you get the error message to unlock your wallet before performing a transaction, that error message needs to say something in plain English like 'Please unlock your wallet before ______'

![image](https://user-images.githubusercontent.com/15713199/27955162-64feefd2-630b-11e7-9086-1018d696ecab.png)

**Comment by dwasyluk** [comment]
Fixed in 1.1.1
